JsonFormReadme.md

```diff
@@ -0,0 +1,62 @@
+
+-----------------------------------------------------------------------------------------------
+# documentation   
+
+## required attributes
+
+1) key - a key is always string and enclosed in quotation mark
+
+2) name - specify the name of a feild
+
+3) label - used to create caption for a form control
+
+4) id - specify the unique id for an element
+
+5) type - defines the feild
+
+6) placeholder - shows information belongs in a feild
+
+7) validation - this value is a function that return a string errort or null if the feild is valid
+
+8) validation server - it needs to validate on the server in addtion to on the client
+
+9) class -
+
+10) wrapper class - Its provide all the of the componnets attribute and properties as props on
+
+the react component
+ 
+
+## validation
+
+1) required - this keeps a list of required properties 
+
+2) min
+        
+3) letters
+
+4) mixedcase
+
+5) numbers
+
+6) uncompromised
```

Comment:
@sajana1829 What is this?

Reply:
these are for password validation.

Reply:
No validation like this was defined in our code.

Reply:
ok